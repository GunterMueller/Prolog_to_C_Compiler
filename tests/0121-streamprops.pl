main :-
	display('current_input:\n'), props(current_input),
	display('current_input:\n'), props(current_output),
	open('tests/0121-streamprops.pl', read, S),
	display('file:\n'), props(S),
	close(S).

props(S) :-
	stream_property(S, P),
	(P = file_no(N), N > 2 -> display(file_no('?')); display(P)),
	nl, fail.
props(_).
