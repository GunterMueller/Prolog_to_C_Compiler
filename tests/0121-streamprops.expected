current_input:
tty(true)
file_no(0)
current_input:
position(51)
tty(false)
file_no(1)
file:
position(0)
tty(false)
file_no(?)
file_name(tests/0121-streamprops.pl)
input
mode(read)
